apps/pay/components/currency/currency-dropdown.tsx

```diff
@@ -17,26 +16,28 @@ export default function CurrencyDropdown({
   showOnlyFlag?: boolean
 }) {
   const searchParams = useSearchParams()
-  const display = searchParams?.get("display")
+  const display = searchParams?.get("displayCurrency")
```

Comment:
I would keep display for now. keep the breaking changes / higher risk element for a smaller PR. so we can isolate the effect of it and decide later if that makes sense or not to do it.

Reply:
I added support for display as well, and it was working with URLs with display parameter. However, I am reverting this to make a smaller PR later.